src/MCControl.cpp

```diff
@@ -143,6 +143,26 @@ MCControl::MCControl(RTC::Manager* manager)
     }
   }
 
+  // confirm mc-rtc timestep is same as IOB timestep
+  if(!m_is_simulation)
```

Comment:
`m_is_simulation` is always `false` even in simulation at this point in the program. Even under `onExecute`, this parameter is false for first few iterations... :thinking:

Reply:
@gergondet Would you know this happens?

Reply:
The setting is set after the constructor. I think you can push the check to [here](https://github.com/jrl-umi3218/mc_openrtm/blob/master/src/MCControl.cpp#L634) because this happens late enough that the `m_is_simulation` member is already correctly set but early enough that nothing critical has been done